internal/pkg/aws/ec2/ec2.go

```diff
@@ -83,14 +84,51 @@ func New(s *session.Session) *EC2 {
 	}
 }
 
-// ListVPC returns IDs of all VPCs.
-func (c *EC2) ListVPC() ([]string, error) {
-	var vpcs []*ec2.Vpc
+// VPC contains the ID and name of a VPC.
+type VPC struct {
+	ID   string
+	Name string
+}
+
+// String formats the elements of a VPC into a display-ready string.
+func (v *VPC) String() string {
+	if v.Name != "" {
+		return fmt.Sprintf("%s (%s)", v.ID, v.Name)
+	}
+	return v.ID
+}
+
+// ExtractVPC extracts the VPC ID from the VPC display string.
+func ExtractVPC(label string) (*VPC, error) {
+	splitVPC := strings.SplitN(label, " ", 2)
+	// TODO: switch to regex
+	switch len(splitVPC) {
+	case 2:
+		if strings.Contains(label, "(") {
```

Comment:
Hmm i feel like for now we can skip this validation since even with this validation we cannot be 100% sure that we parse the name we want, and adding this validation seems doesn't add a lot of values.

Reply:
Ah, yes. I was worried about strings without parentheses, but --of course-- nothing will be removed and the string will just be returned. Will fix.